libs/dal/src/repositories/notification-template/notification-template.entity.ts

```diff
@@ -45,8 +45,6 @@ export class NotificationTemplateEntity {
 
   _notificationGroupId: string;
 
-  _parentId?: string;
```

Comment:
I have gone through entire code and all places steps are used like chain/array and removing this has zero impact, but in cases if you guys think we need to put back we can put back, this has zero impact in new code.

Reply:
@tsssdev @p-fernandez it will have affect. The _parentId of a notification template is the id of template in the environment before. Meaning  a template in production will have _parentId equals to the id that template in development.

Reply:
Sure @ainouzgali I can put that back.